genderbias/publications/__init__.py

```diff
@@ -82,35 +82,24 @@ def __init__(self, **kwargs) -> None:
         super().__init__()
         self.min_publications = kwargs.get("min_publications", 0.5)
 
-    def get_flags(self, doc: "Document") -> List["Flag"]:
+    def get_summary(self, doc: "Document") -> str:
         """
-        Flag a document (globally) if we cannot find any research products.
-
-        Returns only a single flag if no publications/resources are mentioned.
+        Returns a string representing summary feedback on publications.
+        This will indicate if not enough publications were found.
+        Otherwise it will be empty.
         """
-        all_flags = []
-        # TODO: Any other flags needed here?
-
         # Sum up all of the probabilities of all publications. This is a bit
         # janky, but it acts as a proxy for the total number of publications
         # mentioned. For example, if there are two potential publications each
         # with a probability of 50%, then we could consider that a mention of
         # one single publication.
+        summary = ""
         pub_count = sum(identify_publications(doc).values())
         if pub_count < self.min_publications:
-            all_flags.append(
-                Flag(
-                    0,
-                    0,
-                    Issue(
-                        "Publications",
-                        "This document does not mention many publications.",
-                        "Try referencing more concrete publications or work "
-                        "byproducts, if possible.",
-                    ),
-                )
-            )
-        return all_flags
+            summary =   "This document does not mention many publications. " + \
```

Comment:
Nice! I don't think you need both+ as well as / because the operator will indicate a line continuation to the interpreter. (I usually use `black` to format my python files and have been considering adopting it for this codebase if that's interesting to you. Would automatically make things like this consistent with the general python community)

Reply:
Thanks, @j6k4m8! I see [this advice](https://note.nkmk.me/en/python-long-string/) agrees with you. We were getting syntax errors and our Python is a bit rusty. We will fix it.

Reply:
Done! A \ by itself did the job.